catalog/app/utils/Data.js

```diff
@@ -41,13 +47,13 @@ export function useData(request, params, { noAutoFetch = false } = {}) {
       })
   })
 
-  usePrevious({ params, noAutoFetch }, (prev) => {
-    if (R.equals({ params, noAutoFetch }, prev)) return
+  const prev = usePrevious({ params, noAutoFetch })
```

Comment:
`prev` is implementation. Please, use more specific name

Reply:
it means smth like "previous value of the arguments", so it's not "implementation", it's semantically correct